dashboard/client/dashboard.js

```diff
@@ -5,8 +5,74 @@ Template.dashboard.onCreated(function () {
   // Get reference to template instance
   const templateInstance = this;
 
-  // Create reactive variable for Elasticsearch host
+  // Create reactive variable for Elasticsearch host & data
   templateInstance.elasticsearchHost = new ReactiveVar();
+  templateInstance.elasticsearchData = new ReactiveVar();
+
+  const queryParams = {
+    size: 0,
+    body: {
+      query: {
+        filtered: {
+          query: {
+            bool: {
+              should: [
+                {
+                  wildcard: {
+                    request_path: {
+                      // Add '*' to partially match the url
+                      value: '/*',
+                    },
+                  },
+                },
+              ],
+            },
+          },
+        },
+      },
+      aggs: {
+        // Create data range with interval
+        requests_over_time: {
+          date_histogram: {
+            field: 'request_at',
+            interval: 'month',
+          },
+          aggs: {
+            // Get average value of response time for each range
+            avg_response_time: {
+              avg: {
```

Comment:
Here is what I mean when I say "the response time data is skewed":

![screenshot_20170616_180928](https://user-images.githubusercontent.com/17307/27232541-1cf18576-52bf-11e7-805c-0e8ed873283a.png)

Here is a diagram showing what happens to the mean, median, and mode when a data set is skewed versus normal (notice the blue 'mean' line is not close to the middle of the skewed data):

![](https://upload.wikimedia.org/wikipedia/commons/d/de/Comparison_mean_median_mode.svg)